lessons/pydata/api/index.ipynb

```diff
@@ -0,0 +1,1713 @@
+{
```

Comment:
<p>Co říct že tohle je ekvivalentní <code>?lat=51.5487158&amp;lng=....</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

Reply:
je tam koment ...jako lokaci jsem vybral nechvalně proslulý obvod Hackney v Londýně
Nebo myslíš, co je to konkrétně?

Reply:
<p>Já myslel říct že tohle volání s params je ekvivaletní URL</p><pre class="ql-syntax" spellcheck="false">https://data.police.uk/api/stops-street?lat=51.5487158&amp;lng=-0.0613842&amp;date=2018-06
</pre>

Reply:
<p>aha, ok, doplněno</p>